configs/typedoc.json

```diff
@@ -1,7 +1,14 @@
 {
   "ignoreCompilerErrors": true,
-  "mode": "file",
-  "out": "docs/api",
+  "mode": "modules",
+  "out": "gh-pages/docs/next",
   "excludeExternals": true,
-  "includeDeclarations": true
+  "includeDeclarations": true,
+  "readme": "none",
+  "module": "commonjs",
+  "target": "es6",
+  "hideGenerator": true,
+  "external-modulemap": ".*\/packages\/([\\w\\-_]+)\/",
+  "name": "Theia TypeDoc",
+  "exclude": "**/+(dev-packages|examples|typings)/**/*.ts"
```

Comment:
Could we add one more exclude to filter out spec files:
```json
"exclude": [
    "**/+(dev-packages|examples|typings)/**/*.ts",
    "**/*spec.ts"
  ]
```

Reply:
I'm not sure it works with this format of exclude ? but I may be wrong of course

Reply:
It seems to be valid syntax: https://github.com/TypeStrong/typedoc/issues/170#issuecomment-388607358

Reply:
ok nice 👍